test/nlp/test_autohf_maxiter1.py

```diff
@@ -41,15 +41,15 @@ def toy_metric(
         groups_train=None,
     ):
         return 0, {
-            "test_loss": 0,
+            "val_loss": 0,
             "train_loss": 0,
             "pred_time": 0,
         }
 
     automl_settings = {
         "gpu_per_trial": 0,
-        "max_iter": 1,
-        "time_budget": 5,
+        "max_iter": 3,
```

Comment:
Not matching the file name.
Please provide justification why we want to use a longer test.